src/applications/vaos/tests/appointment-list/components/cards/confirmed/VideoVisitProvider.unit.spec.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { expect } from 'chai';
+import { render } from '@testing-library/react';
+import VideoVisitProvider from '../../../../../appointment-list/components/cards/confirmed/VideoVisitProvider';
+
+const participantMock = [
+  { actor: { reference: 'Location/test' } },
+  { actor: { reference: 'Practitioner/Tester', display: 'Tester' } },
+];
+
+describe('VAOS integration: VA Video Visit Provider', () => {
```

Comment:
Like we talked about last week, I think we can skip this test and modify one of the video appointment tests in the FutureAppointmentsList tests to show providers.

Reply:
Fixed